pkg/driver/node_server.go

```diff
@@ -34,6 +35,10 @@ var (
 	ephemeralUnpublishLock sync.Mutex
 )
 
+// Maximum default number of volumes that controller can publish to the node.
+const defaultMaxVolPerNode = 100
+const maxVolumePerNodeKey = "MAX_VOLUMES_PER_NODE"
+
```

Comment:
We have a constants.go file where we keep these.

Reply:
Sorry I didn't notice. I will use it.